content/gsoc/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "GNU Radio in the Google Summer of Code 2019"
+type: gsoc
+draft: true
+---
+
+## Google Summer of Code 2019
```

Comment:
Add more blurb and link to nested GSoC pages? Or should the starting point be `/gsoc/ideas/` anyway?